async_task.js

```diff
@@ -0,0 +1,88 @@
+/* 1. Створити модуль api_helper.js, в якому будуть реалізовані функції посилання get та post запитів
+та десеріалізація респонсу у json об'єкт, який вони будуть повертати (при реалізації поста зверніть увагу на хедери, 
+які вказані у інструкції під блоком Post with JSON). Для створення запитів використати пакет
+node-fetch, інструкця по використанню тут https://www.npmjs.com/package/node-fetch?activeTab=readme, для 
+реалізації асинхронності використати async/await */
+
+import * as request from "./api_helper/api_helper.js";
+
+const baseUrl = "https://jsonplaceholder.typicode.com/posts/";
+
+
+// /* 2. Реалізувати функцію getPostsByUsedId, яка буде знаходити пости з ресурсу https://jsonplaceholder.typicode.com/posts та робити 
+// вибірку постів за id користувача. Також у всіх відфільтрованих постів повинна бути відсутня властивість title */
+
+// async function getPostsByUsedId(url, userId) {
+// // Ваш код
+
+// try {
+//     const getResponse = await fetch(url);
```

Comment:
```suggestion
//     const getResponse = await request.getRequest(url);
```
варто було перевикористати створену функцію з api_helper

Reply:
Розкоментуй свою реалізацію, чому вона закоментована?

Reply:
@Art-auto 
замінив на 19й рядок на 
const getResponse = await request.getRequest(url);

АЛЕ тепер помилка в консолі

![image](https://github.com/qarber/hillel_qa_automation/assets/147742705/7cbc4402-b986-4e2f-ab8a-2d25b1c6b986)